.rubocop.yml

```diff
@@ -0,0 +1,12 @@
+AllCops:
+  Exclude:
+    - 'Rakefile'
+    - 'db/seed.rb'
+  TargetRubyVersion: 2.6.3
+  DisplayCopNames: true
+
+Style/Documentation:
```

Comment:
Why not having docs? For OSS projects it makes a lot of sense. And there is always `# :nodoc:`)